docs/administration-guide/gaffer-deployment/gremlin.md

```diff
@@ -28,27 +28,21 @@ traversals are spawned. To do this we recommend utilising the provided
 which can be configured to use the Gaffer Tinkerpop implementation so that a
 endpoint is available for Gremlin queries.
 
-## Connecting to Any Existing Gaffer Graph
+## Connecting to An Existing Accumulo Backed Graph
 
-The simplest way to connect Gremlin to an existing Gaffer instance where you may
-not know the Store type or Schema would be via a [Proxy Store](../gaffer-stores/proxy-store.md).
-Connecting this way means Gremlin communicates via the Gaffer REST API
-(similar to [gafferpy](../../user-guide/apis/python-api.md)) meaning there may
-be a performance hit for larger queries.
-
-!!! tip
-    You can also of course connect directly to an existing instance's storage
-    layer too (e.g. Accumulo store) but this would require a more complex
-    configuration and knowledge of the Schema.
+The recommended way to provide a Gremlin interface to an existing Gaffer
+instance is to connect directly to the same [Accumulo store](../gaffer-stores/accumulo-store.md).
```

Comment:
I think we should mention that it is possible to connect to other store types and that the steps will be similar with only accumulo specific aspects changing.